pip_faster.py

```diff
@@ -73,7 +73,7 @@ def either(char):
 
             from pkg_resources import parse_version
             version = parse_version(wheel.version)
-            if version > best_version:
+            if best_version is None or version > best_version:
```

Comment:
Let's use `parse_version('')` instead of None.
This is produces a parsed version tuple of `(-1, ('*final',))` which will compare less-than any other (sane) version.